### PR TITLE
created Readme.md in My Amigo - html

### PR DESCRIPTION
**Pull request by Anuradha0501**
## Pull Request Checklist

Is your project an
- [ ] Extension 
- [x] Application

Given below are the mandatory requirements we expect from your project. Your Pull Request will be reviewed only if all the minimum requirements relevant to your project are satisfied.

- [x] Added readme.md to my project folder with relevant information.
- [x]  I have provided the demo video link (showing the functionality in action) or the public deployment link of my extension of application within the readme.md (if Applicable).
- [x] I have explained the process in a readable manner within readme.md.

### Terms and Conditions

`Plagiarism check from Team Crio will be solid and if your code is found to be plagiarized, the team has every right to reject your Pull Request and take necessary actions.`
- [x] I can assure you that this is my own contribution and I did not involve in any kind of plagiarism activities.

### CODE REVIEW

**Comment by sudhanshutiwari264** [comment]
Thanks for your contribution @Anuradha0501 🎉
Kindly edit your pr template 2nd part 1st point check ✅ !
Please wait our team is reviewing this !

**Comment by Anuradha0501** [comment]
> Thanks for your contribution @Anuradha0501 🎉
> Kindly edit your pr template 2nd part 1st point check ✅ !
> Please wait our team is reviewing this !

Edited

**Review by jainaayush01 — CHANGES_REQUESTED** [review]
Great work with the project.
Changes:
1. your video demo only showcase one command can you make a detailed video with all major commands your assistant performs
2. the giphyAPIkey is available public, not a good practice.

cheers

**Comment by Anuradha0501** [comment]
> Great work with the project.
> Changes:
> 
> 1. your video demo only showcase one command can you make a detailed video with all major commands your assistant performs
> 2. the giphyAPIkey is available public, not a good practice.
> 
> cheers

okay i will make first change but i dont know alternative for giphyAPIkey as i am beginner right now .

File: My Amigo - html/meta/templates/speech-recognition-to-gif.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+<meta content="ie=edge" http-equiv="X-UA-Compatible">
+<title>Speak a Topic - View a Gif</title>
+<meta name="description" content="Demonstration of SpeechRecognition API that responds to voice commands and displays random animated gifs.">
+<!-- minimum viable  social tags -->
+<meta property="og:title" content="Speak a Topic - View a Gif"/>
+<meta property="og:description" content="Prototypes that demonstrate voice user interfaces for libraries."/>
+<meta property="og:image" content="http://www.lib.montana.edu/~jason/meta/img/clark-share-default.png"/>
+<meta property="og:url" content="http://www.lib.montana.edu/~jason/files/voice-user-interface/"/>
+<meta property="og:type" content="website"/>
+<meta name="twitter:creator" property="og:site_name" content="@jaclark"/>
```
Comment (jainaayush01) [inline My Amigo - html/meta/templates/speech-recognition-to-gif.html:15]: Very sad to see this, 
kindly close this PR, we do not tolerate plagiarism.

Comment (Anuradha0501) [reply My Amigo - html/meta/templates/speech-recognition-to-gif.html:15]: if i remove these meta tags then it will be ok

Comment (Anuradha0501) [reply My Amigo - html/meta/templates/speech-recognition-to-gif.html:15]: if not then i will close this pr

**Pull request closed without merging**